synapse/lib/cell.py

```diff
@@ -1027,9 +1028,10 @@ async def initServiceStorage(self):
         pass
 
     async def initNexusSubsystem(self):
-        mirror = self.conf.get('mirror')
-        await self.nexsroot.startup(mirror, celliden=self.iden)
-        await self.setCellActive(mirror is None)
+        if self.cellparent is None:
+            mirror = self.conf.get('mirror')
+            await self.nexsroot.startup(mirror, celliden=self.iden)
+            await self.setCellActive(mirror is None)
```

Comment:
We never call `self.initServicePassive()` with the current delta.

Reply:
Yep, that's def included in the ticket i created which noted that we will need to eventually figure out controlling an embedded cell's active/passive callbacks and hamstring nexsoffset manipulation (required if we ever want to expose an embedded cell via telepath ).  Future self! :)